src/doom/hu_stuff.c

```diff
@@ -1073,6 +1085,7 @@ void HU_Ticker(void)
     {
 	crispy_statsline_func_t crispy_statsline = crispy_statslines[crispy->statsformat];
 
+	if (crispy->automapstats == WIDGETS_STBAR)
```

Comment:
Isn't this check redundant at this place?

Reply:
Redundant indeed, fixed in commit below.

Reply:
Guess it's all done now, except some unclear positioning between tabs and spaces.
@fabiangreffrath are there any further recommendations?

Reply:
Sorry, I'll still have to play-test the changes, but haven't found time for it yet, because I am currently involved in other projects.